crates/core/tedge/src/bridge/aws.rs

```diff
@@ -101,7 +98,7 @@ fn test_bridge_config_from_aws_params() -> anyhow::Result<()> {
         cloud_name: "aws".into(),
         config_file: "aws-bridge.conf".to_string(),
         connection: "edge_to_aws".into(),
-        address: "test.test.io:8883".into(),
+        address: HostPort::<MQTT_TLS_PORT>::try_from("test.test.io")?,
```

Comment:
Since the HostPort is doing some additional logic (e.g. selecting a default port etc.), it might make sense to have this test check the value against the expected string output (e.g. `test.test.io:8883` rather than just comparing the HostPort struct)

Reply:
I got the exactly same thought, also on other cloud mappers, az and c8y.

Reply:
Added unit tests at the source level itself for `HostPort.to_string()`, to avoid duplication of the same for all clouds.